### PR TITLE
Update for Chef InSpec Resource Code Examples

### PR DESCRIPTION
**Pull request by mjingle**
Signed-off-by: Mary Jinglewski <mjinglewski@chef.io>

## Description

This PR updates Chef InSpec documentation specifically for multi-line code examples for ntp_conf and sshd_config. This content was sourced from the original code example and then cookstyle was applied to it.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New content (non-breaking change)
- [ ] Breaking change (a content change which would break existing functionality or processes)

## Checklist:
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: docs/resources/ntp_conf.md.erb [hunk 1]
```diff
@@ -43,11 +43,13 @@ The following examples show how to use this Chef InSpec audit resource.
 
     describe ntp_conf do
       its('driftfile') { should eq '/var/lib/ntp/ntp.drift' }
-      its('server') { should eq [
+      its('server') do
+        should eq [
         0.ubuntu.pool.ntp.org,
```
Comment (clintoncwolfe) [inline docs/resources/ntp_conf.md.erb]: Shouldn't these be quoted?
Also, likely indented two more spaces?

**Review by clintoncwolfe — CHANGES_REQUESTED** [review]
I think the ntp_conf existing code may have been bad - it may have needed quotes.

**Review by miah — CHANGES_REQUESTED** [review]
re: clintons feedback =)

File: docs/resources/ntp_conf.md.erb (hunk 1 above)
Comment (mjingle) [reply docs/resources/ntp_conf.md.erb]: Thanks for catching this! Fixing now.

**Comment by mjingle** [comment]
> I think the ntp_conf existing code may have been bad - it may have needed quotes.

@clintoncwolfe Is there more work needed to be done on the ntp_conf existing code then?

File: docs/resources/ntp_conf.md.erb [hunk 2]
```diff
@@ -43,11 +43,13 @@ The following examples show how to use this Chef InSpec audit resource.
...
     describe ntp_conf do
       its('driftfile') { should eq '/var/lib/ntp/ntp.drift' }
-      its('server') { should eq [
-        0.ubuntu.pool.ntp.org,
-        1.ubuntu.pool.ntp.org,
-        2.ubuntu.pool.ntp.org
-      ] }
+      its('server') do
+        should eq [
+          0.ubuntu.pool.ntp.org,
```
Comment (miah) [inline docs/resources/ntp_conf.md.erb]: '0.ubuntu.pool.ntp.org',

Comment (mjingle) [reply docs/resources/ntp_conf.md.erb]: Thank you! Fixed through this code block.

**Review by clintoncwolfe — APPROVED** [review]
Looks good, thanks!

**Review by miah — APPROVED** [review]

**Review by zenspider — APPROVED** [review]

**Pull request merged**